packages/javascript-mutator/src/mutators/StringLiteralMutator.ts

```diff
@@ -7,27 +7,19 @@ import { NodeMutator } from './NodeMutator';
 export default class StringLiteralMutator implements NodeMutator {
   public name = 'StringLiteral';
 
-  public mutate(node: NodeWithParent, copy: <T extends types.Node>(obj: T, deep?: boolean) => T): types.Node[] {
-    const nodes: types.Node[] = [];
-
-    if (types.isTemplateLiteral(node)) {
-      nodes.push({
-        end: node.end,
-        innerComments: node.innerComments,
-        leadingComments: node.leadingComments,
-        loc: node.loc,
-        start: node.start,
-        trailingComments: node.trailingComments,
-        type: 'StringLiteral',
-        value: node.quasis.length === 1 && node.quasis[0].value.raw.length === 0 ? 'Stryker was here!' : ''
-      } as types.StringLiteral);
-    } else if ((!node.parent || this.isDeclarationOrJSX(node.parent)) && types.isStringLiteral(node)) {
-      const mutatedNode = copy(node);
-      mutatedNode.value = mutatedNode.value.length === 0 ? 'Stryker was here!' : '';
-      nodes.push(mutatedNode);
-    }
-
-    return nodes;
+  public mutate(node: NodeWithParent): Array<[types.Node, types.Node | { raw: string }]> {
+    return types.isTemplateLiteral(node)
```

Comment:
again too many shorted ifs

Reply:
Done for this mutator.

It was not clear to me whether or not we need to go back to if statements in all of the other mutators. I am fine to do this but would like to see PR #1903 reviewed and merged first.